<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建您的账户</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(33, 36, 41);
color:rgb(255, 255, 255);
font: 8pt &quot;微软雅黑&quot;;
}

QLineEdit{
	font: 10pt &quot;微软雅黑&quot;;
	padding:10px;
	background-color: rgb(50, 53, 60);
	border: 0px;
}

QPushButton{
	background-color: rgb(118, 119, 121);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;欢迎注册&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>382</width>
     <height>48</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入手机号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#pushButton_register{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(6, 191, 255), stop:1 rgb(44, 114, 255) );
	color: rgb(255, 255, 255);
	border-radius:3px;		
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QPushButton#pushButton_register:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}

QPushButton#pushButton_register:pressed{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:0.5 rgb(5, 191, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;	
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>381</width>
     <height>48</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码8-20位，包括数字/字母</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>381</width>
     <height>48</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>再次输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>306</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
